website/content/en/preview/reference/metrics.md

```diff
@@ -202,6 +226,10 @@ Total number of reconciliations per controller
 Length of time per reconciliation per controller
 - Stability Level: STABLE
 
+### `controller_runtime_reconcile_panics_total`
+Total number of reconciliation panics per controller
+- Stability Level: STABLE
+
```

Comment:
Do all of our controllers already recover from panics and not crash the manager?

Reply:
Hmmmm -- yeah it looks like controller_runtime added this metric in the latest release. I didn't even know this existed until you just called this out.